packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -255,11 +258,6 @@ export interface IDetailsListProps extends IBaseProps<IDetailsList>, IWithViewpo
    */
   enterModalSelectionOnTouch?: boolean;
 
-  /**
-   * On horizontal scroll event listener
-   */
-  onScroll?: (e?: Event) => void;
```

Comment:
Is this a breaking API change? Should this have been marked deprecated? It appears we still have some examples using onScroll...